pkg/onchain/sc.go

```diff
@@ -119,9 +119,21 @@ func DeploySC(client *node.Client,
 	signer signer.Signer,
 	description string,
 ) (*sendOperation.OperationResponse, []node.Event, error) {
+	// Calibrate max_gas
+	if maxGas == sendOperation.DefaultGasLimitExecuteSC || maxGas == 0 {
```

Comment:
this condition is strange, if we dont want to use DefaultGasLimitExecuteSC, we should delete it and do the estimation only when maxgas is not provided